src/main/java/eu/europeana/metis/sandbox/entity/DatasetEntity.java

```diff
@@ -36,11 +36,20 @@
   @Enumerated(EnumType.STRING)
   private Country country;
 
+  private String xsltTransformerEdmExternal = "";
```

Comment:
Why initializing here?
Also this is not a transformer but rather the contents of the xslt right?

Reply:
I don't remember why I did it, I will remove the initialization.
The name is also a good point, I will change that